nebula/ui/components/MZInformationCard.qml

```diff
@@ -51,8 +119,11 @@ Rectangle {
         }
 
         MZIcon {
-            source: "qrc:/nebula/resources/info.svg"
-            Layout.alignment: Qt.AlignTop
+            source: getIconImage()
+            Layout.maximumHeight: 18
```

Comment:
It looks like the `warning-gray.svg` is smaller/has less surrounding white space than `success.svg` which I think means `success.svg` will look smaller when constrained to a max height/width of 18. If we change the warning icon code to the snippet below, which includes the same surrounding white space as `success.svg`, we should be able to drop these two sizing lines (Layout.maximumHeight, Layout.maximumWidth) and have the icons appear more uniformly sized. 
```
<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M18.6929 16.0119L13.7399 6.10586C13.5739 5.77358 13.3185 5.49411 13.0025 5.29878C12.6866 5.10346 12.3224 5 11.9509 5C11.5795 5 11.2153 5.10346 10.8993 5.29878C10.5834 5.49411 10.328 5.77358 10.1619 6.10586L5.20894 16.0159C5.0575 16.3206 4.98633 16.659 5.00217 16.999C5.01801 17.3389 5.12033 17.6692 5.29946 17.9586C5.47858 18.2479 5.72858 18.4868 6.0258 18.6526C6.32302 18.8183 6.65762 18.9055 6.99794 18.9059H16.9029C17.2439 18.906 17.5792 18.819 17.8771 18.6531C18.1749 18.4872 18.4255 18.2479 18.6048 17.9579C18.7842 17.668 18.8864 17.337 18.9018 16.9964C18.9172 16.6557 18.8453 16.3169 18.6929 16.0119ZM10.9509 8.90586C10.9509 8.64065 11.0563 8.38629 11.2438 8.19876C11.4314 8.01122 11.6857 7.90586 11.9509 7.90586C12.2162 7.90586 12.4705 8.01122 12.658 8.19876C12.8456 8.38629 12.9509 8.64065 12.9509 8.90586V12.9059C12.9509 13.1711 12.8456 13.4254 12.658 13.613C12.4705 13.8005 12.2162 13.9059 11.9509 13.9059C11.6857 13.9059 11.4314 13.8005 11.2438 13.613C11.0563 13.4254 10.9509 13.1711 10.9509 12.9059V8.90586ZM11.9509 17.1559C11.7037 17.1559 11.462 17.0826 11.2565 16.9452C11.0509 16.8078 10.8907 16.6126 10.7961 16.3842C10.7015 16.1558 10.6767 15.9045 10.725 15.662C10.7732 15.4195 10.8922 15.1968 11.0671 15.022C11.2419 14.8472 11.4646 14.7281 11.7071 14.6799C11.9496 14.6317 12.2009 14.6564 12.4293 14.751C12.6577 14.8456 12.8529 15.0058 12.9903 15.2114C13.1276 15.417 13.2009 15.6586 13.2009 15.9059C13.2009 16.2374 13.0692 16.5553 12.8348 16.7897C12.6004 17.0242 12.2825 17.1559 11.9509 17.1559Z" fill="#3D3D3D"/>
</svg>

Reply:
Thanks, updated this. Images look better

<img width="355" alt="Screenshot 8" src="https://github.com/mozilla-mobile/mozilla-vpn-client/assets/9295855/decc3225-50fa-4a45-832c-ed31c89a21e1">
<img width="353" alt="Screenshot 9" src="https://github.com/mozilla-mobile/mozilla-vpn-client/assets/9295855/2e75942f-6367-432f-b692-fabe9a31da46">
<img width="359" alt="Screenshot" src="https://github.com/mozilla-mobile/mozilla-vpn-client/assets/9295855/2a0e3dbd-f6f1-49ff-a130-b4a793a75e68">
<img width="350" alt="Screenshot 10" src="https://github.com/mozilla-mobile/mozilla-vpn-client/assets/9295855/8506c10f-f47b-4f85-ac69-057518ae816d">